projects/Apps/scripts/package.json

```diff
@@ -6,6 +6,11 @@
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1"
     },
+    "dependencies": {
+        "aws-sdk": "^2.511.0",
```

Comment:
if at some point in the future we want a smaller artifact we could use https://github.com/Junkern/modular-aws-sdk instead of the full 40mb sdk. Makes sense to stick with the AWS one for now though!